packages/jobs/lib/crons/autoIdleDemo.ts

```diff
@@ -37,15 +36,26 @@ export function cronAutoIdleDemo(): void {
     });
 }
 
-export async function exec(): Promise<void> {
+export async function exec({ orchestrator }: { orchestrator: Orchestrator }): Promise<void> {
     logger.info('[autoidle] starting');
 
-    const syncs = await findPausableDemoSyncs();
+    const syncs = await findDemoSyncs();
```

Comment:
It is bit more complicated since schedules are not in the same database anymore and we cannot simply join syncs with schedules tables to find out which ones are running. We could limit to only search the syncs updated in the last X days so the size doesn't grow indefinitely overtime, assuming this cron runs successfully at least once during this period. Can you think of a better idea?

Reply:
yes make sense, not ideal but will scale for a long time

Reply:
A long term solution would be to add a `pause_after` feature into the orchestrator. For now I will keep the cron job